### PR TITLE
Make the save dialog refer to the save all action, not the save action

### PR DESCRIPTION
**Pull request by DaveShuckerow**
We had a user who was confused when he ran 'save document' and didn't get a hot reload.

### CODE REVIEW

File: src/io/flutter/FlutterBundle.properties [hunk 1]
```diff
@@ -219,7 +219,7 @@ settings.format.code.on.save=Format code on save
 settings.organize.imports.on.save=Organize imports on save
 settings.flutter.version=Version:
 settings.open.inspector.on.launch=Open Flutter Inspector view on app launch
-settings.hot.reload.on.save=Perform hot reload on save
+settings.hot.reload.on.save=Perform hot reload on Save All
```
Comment (devoncarew) [inline src/io/flutter/FlutterBundle.properties]: Can we instead keep this as `save` (saving generally, and not the specific IntelliJ action), and call out the `Save All` action in tooltip text?

Comment (DaveShuckerow) [reply src/io/flutter/FlutterBundle.properties]: SGTM.

File: src/io/flutter/sdk/FlutterSettingsConfigurable.form [hunk 2]
```diff
@@ -121,7 +121,7 @@
             </constraints>
             <properties>
               <text resource-bundle="io/flutter/FlutterBundle" key="settings.hot.reload.on.save"/>
-              <toolTipText value="On save, hot reload changes into running Flutter apps."/>
+              <toolTipText value="On save all, hot reload changes into running Flutter apps."/>
```
Comment (devoncarew) [inline src/io/flutter/sdk/FlutterSettingsConfigurable.form]: `On a 'Save All' action, hot reload...`?

Comment (DaveShuckerow) [reply src/io/flutter/sdk/FlutterSettingsConfigurable.form]: Done.

**Pull request merged**